app/notification/application/map_contents.py

```diff
@@ -157,3 +198,119 @@ def bytes_object_for_questions_answers(
+        #     2. Second item
+        """
+        try:
+            if answer is None or isinstance(answer, (bool, list)):
+                return answer
+
+            soup = BeautifulSoup(answer, "html.parser")
+            indent = " " * 5
+
+            if soup.find():
+                if soup.ul:
+                    bullet_points = []
+                    for index, li in enumerate(
+                        soup.ul.find_all("li"), start=1
+                    ):
+                        if li.get_text() != "\xa0":
+                            if index > 1:
+                                bullet_points.append(
+                                    f"{indent}- {li.get_text()}"
+                                )
+                            else:
+                                bullet_points.append(f"- {li.get_text()}")
+                    return "\n".join(bullet_points)
+                elif soup.ol:
+                    numbered_list = []
+                    for index, li in enumerate(
+                        soup.ol.find_all("li"), start=1
+                    ):
+                        if li.get_text() != "\xa0":
+                            if index > 1:
+                                numbered_list.append(
+                                    f"{indent}{index}. {li.get_text()}"
+                                )
+                            else:
+                                numbered_list.append(
+                                    f"{index}. {li.get_text()}"
+                                )
+                    return "\n".join(numbered_list)
+                else:
+                    # Handle other HTML tags
+                    cleaned_text = soup.get_text()
+                    cleaned_text = cleaned_text.replace("\xa0", "")
+                    return cleaned_text.strip()
+            else:
+                return answer.strip()
+
+        except Exception as e:
+            current_app.logger.error(
+                f"Error occurred while processing HTML tag: {answer}", e
+            )
+
+            return answer
+
+    @classmethod
+    def sort_multi_input_data(cls, multi_input_data):
+        """
+        Sorts and formats multi-input data.
+
+        Args:
+            multi_input_data (list[dict]): A list of dictionaries representing multi-input data.
+            example: [{'GLQlOh': 'cost one', 'JtwkMy': 4444}, {'GLQl6y': 'cost two', 'JtwkMt': 4455}]
+
+        Returns:
+            str: A formatted string representation of the sorted multi-input data.
+            example: A) - cost one: £4444
+                        - cost two: £4455
+        """
+        key = None
+        value = None
+        sorted_data = {}
+        indent = " " * 5
+
+        for items in multi_input_data:
+            for index, v in enumerate(items.items()):
+                if index == 0:
+                    key = v[-1]
+                if index == 1:
+                    value = v[-1]
+                sorted_data[key] = value
```

Comment:
What about using `.values()` instead of `.items()` and unpacking the tuple?

```suggestion
           key, value = items.values()
           sorted_data[key] = value
```

```python
>>> key, value = {'GLQlOh': 'cost one', 'JtwkMy': 4444}.values()
>>> print(f"{key=}, {value=}")
key='cost one', value=4444
```

I think that's what's happening here?

Reply:
Yes, you are right. I had a different approach to it at first. We could do it with just values().